### DISCUSSION TITLE
Nextcloud-aio fill up docker.img on unraid

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [nextcloud/all-in-one#3329] [Questions] [answered]
### Steps to reproduce
1. Install nextcloud-aio-mastercontainer
2. Upload files from android app

### Expected behavior 
Nextcloud should store data on my data folder in unraid array

### Actual behavior 
Nextcloud upload files in docker.img and fill it up to 100%

### Host OS 
Unraid 6.12.4

#### Nextcloud AIO version 
Nextcloud AIO v7.0.0

#### Current channel 
Lastest

![image](https://github.com/nextcloud/all-in-one/assets/25219988/333e481a-e2d8-40d6-9d67-7786fc57fcd8)

Regards.

**Comment by szaimen** [comment, marked as answer]
Hi, did you configure NEXTCLOUD_DATADIR?
